indico/web/client/styles/modules/categories/_category_statistics.scss

```diff
@@ -0,0 +1,28 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+.ui.segments > .segment {
+    width: 50%;
+}
+
+.chart-container {
+    width: 480px;
```

Comment:
Hm, it is not very flexible if we take into account resizing of the browsers etc.

Reply:
I understand that. If I don't specify the width and height here specifically, the SVG inside the chart will have a `0` height and not be visible. It is the requirement to pass the width and height to div containing the chart.

Reply:
Can't you use some relative units?

Reply:
Done !